install_template/templates/products/mysql-foreign-data-wrapper/debian-11.njk

```diff
@@ -9,8 +9,8 @@
 
   #Enable the MySQL repo:
   For MySQL 8:
-  sudo echo "deb http://repo.mysql.com/apt/debian/buster mysql-8.0" | sudo tee  /etc/apt/sources.list.d/mysql.list
+  sudo echo "deb http://repo.mysql.com/apt/debian/bullseye mysql-8.0" | sudo tee  /etc/apt/sources.list.d/mysql.list
   For MySQL 5:
```

Comment:
Please remove MySQL 5 section as it's not supported on Debian 11

Reply:
removed